Automattic/pocket-casts-ios
# Eoy2024: Fix footers text alignment

| 📘 Part of: # |  
|:---:|

Fixes #2389 #2394 #2395 #2397

| 1 | 2 | 3 | 4 | 5 |
| - | - | - | - | - |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 13 24](https://github.com/user-attachments/assets/edb05d78-7b99-427a-b63c-7a143373fa63) | ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 13 06](https://github.com/user-attachments/assets/d35e4dd7-e653-40c1-aadd-af1dc17b0b9a) | ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 11 53](https://github.com/user-attachments/assets/e51553ca-7559-49c9-a6d9-0aa4b7e4049f) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 11 50](https://github.com/user-attachments/assets/04e78538-17e4-44e8-a202-9d1680700343) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 11 48](https://github.com/user-attachments/assets/4f23699b-8385-4b65-a39d-f7deba347715) | 
| ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 11 46](https://github.com/user-attachments/assets/67fd308b-09b2-45ae-ac6f-f5f9ad6fbf05) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 11 44](https://github.com/user-attachments/assets/0851d4da-6bd9-49bd-ac5b-b0e17c52c5c6) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 11 42](https://github.com/user-attachments/assets/9dde91a7-390b-4732-b6fe-b76baffd11f0) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 11 40](https://github.com/user-attachments/assets/4bcf98e9-673f-4802-ad9e-ca9dd40d7f0b) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 14 11 37](https://github.com/user-attachments/assets/f5f1f57d-6e30-449a-bc66-3c7fcb040564) |

This PR updates the footer view on all stories to share the same code, alignment, fonts.

It also makes two small changes:

- Updates the text of the comparison screen when is more than 500% percent 
- Fixes the position of the listening time large number, by offsetting from the bottom.

I tried to also fix the details font on the footer but after trying all different combination of weights on the font I couldn't exactly match what is on the Figma designs.

## To test

1. Start the app
2. Ensure that you have the EOY 2024 FF active. ( Restart after enabling it)
3. Tap on the EOY banner on profile
4. Check the screens to see if the footer banner is always aligned


## Checklist
